test/index.test.ts

```diff
@@ -206,7 +206,10 @@ describe('FiatConnect SDK', () => {
         'https://fiat-connect-api.com/auth/login',
         expect.objectContaining({
           method: 'POST',
-          headers: { 'Content-Type': 'application/json', Authorization: 'Bearer api-key' },
+          headers: {
```

Comment:
linter did this :eyes:

Reply:
nit: ideally we'd do this in separate PRs

Reply:
https://github.com/fiatconnect/fiatconnect-sdk/pull/16